chain/stmgr/supply.go

```diff
@@ -303,11 +303,14 @@ func getFilPowerLocked(ctx context.Context, st *state.StateTree) (abi.TokenAmoun
 	return pst.TotalLocked()
 }
 
-func GetFilLocked(ctx context.Context, st *state.StateTree) (abi.TokenAmount, error) {
-
-	filMarketLocked, err := getFilMarketLocked(ctx, st)
-	if err != nil {
-		return big.Zero(), xerrors.Errorf("failed to get filMarketLocked: %w", err)
+func GetFilLocked(ctx context.Context, st *state.StateTree, height abi.ChainEpoch) (abi.TokenAmount, error) {
```

Comment:
Please refactor this to receive a network version instead, and make the change based on that. I know we have a lot of height comparisons in this area of the code, but that's quite bad and we shouldn't be adding any more.

The `StateManager` can lookup network versions based on the provided height and supply it here.